docs/rhoas/rhoas-cli-installation/README.adoc

```diff
@@ -230,6 +230,7 @@ You can enable command completion for each of the following shells:
 .Prerequisites
 
 * You've installed the `rhoas` CLI.
+* To enable command completion on Bash for macOS you need to have Bash completion installed and running with its current login scripts.
```

Comment:
```suggestion
* If you're using macOS, `bash-completion` must be installed and configured in your `.bash_profile` file.
```

Reply:
Just one further suggestion here. I would try to write this in the active voice, to match the prerequisite that comes before it. So, something like:

"If you're using macOS, you've installed `bash-completion` and configured this in your `.bash_profile` file."